notebooks/nightly_report/nightly_report_ts_version_simplified_only.ipynb

```diff
@@ -1,4 +1,4 @@
~{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #72.    # -- Right column: DoF plots with shared x-axis (not y)
</pre><p>To be honest, it would be much easier to read if  all subplots were sharing x-axis (seqNum) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I disagree here. When sharing the plots they fit better side-by-side for example when sharing on the screen in this morning's meeting. Also, keeping related measurements separate makes it easier to cut them out and put them into a slide.

Reply:
<p>i agree with bryce here</p>